content/academy/anti_scraping/proxies/using_proxies.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Using Proxies
+description: Learn how to use and automagically rotate proxies in your scrapers by using the Apify SDK.
+menuWeight: 1
+category: courses
+paths:
+- anti-scraping/proxies/using-proxies
+---
+
+# [](#using-proxies) Using Proxies
+
+In the [Web-Scraping for Beginners]({{@link web_scraping_for_beginners.md}}) course, we learned about the power of the Apify SDK, and how it can streamline the development process of web-crawlers. You've already seen how powerful the `apify` package is; however, what you've been exposed to thus far is only the tip of the iceberg.
+
+Because proxies are so widely used in the scraping world, we at Apify have equipped our SDK with features which make it easy to implement them in an effective way. One of the main functionalities that comes baked into the SDK is proxy rotation, which is when each request is sent through a different proxy from a proxy-pool.
+
+## [](#implementing-proxies) Implementing proxies in a scraper
+
+Let's borrow some scraper code from the end of the [pro-scraping]({{@link web_scraping_for_beginners/crawling/pro_scraping.md}}) lesson in our _Web Scraping for Beginners_ course and paste it into a new file called `proxies.js`. This code enqueues all of the product links on [demo-webstore.apify.org](https://demo-webstore.apify.org)'s on-sale page, then makes a request to each product page and scrapes data about each one:
```

Comment:
Please prefer bold over italics for consistency and use code markers only for code stuff ❤️ 

```suggestion
Let's borrow some scraper code from the end of the [pro-scraping]({{@link web_scraping_for_beginners/crawling/pro_scraping.md}}) lesson in our **Web Scraping for Beginners** course and paste it into a new file called **proxies.js**. This code enqueues all of the product links on [demo-webstore.apify.org](https://demo-webstore.apify.org)'s on-sale page, then makes a request to each product page and scrapes data about each one:
```